thirdeye/thirdeye-frontend/app/pods/components/composite-anomalies/parent-anomalies/component.js

```diff
@@ -0,0 +1,120 @@
+/*
+ * Parent Anomalies Component
+ *
+ * Display a table which contains composite anomalies
+ * @module composite-anomalies/parent-anomalies
+ * @property {string} title   - Heading to use on the table
+ * @property {object[]} data  - [required] array of composite anomalies objects
+ *
+ * @example
+ * {{composite-anomalies/parent-anomalies title=<title> data=<data>}}
+ *
+ * @exports composite-anomalies/parent-anomalies
+ */
+
+import Component from '@ember/component';
+import { computed } from '@ember/object';
+
+import moment from 'moment';
+import config from 'thirdeye-frontend/config/environment';
+
+let data = [];
+
+/*
+ *  convert animaly 'start' and 'end' into an object to be used by template: 'custom/composite-animalies-table/start-duration'
+ */
+const getAnomaliesStartDuration = (start, end) => {
+  return {
+    startTime: start,
+    endTime: end,
+    duration: moment.duration(end - start).asHours() + ' hours'
+  };
+};
+
+/*
+ *  convert list of anonalies object in to a objects
+ */
+const getAnomaliesDetails = (details) => {
+  const detailsObject = [];
+  Object.entries(details).forEach((detail) => {
+    detailsObject.push({ name: detail[0], count: detail[1] });
```

Comment:
it's better to omit the undefined value.

Reply:
what do you mean?

Reply:
My concerns is that when detail is a empty array. the detail[0], detail[1] or both are undefined. 
```suggestion
    [[2,3],[],[3,4]].forEach(e=>{
     console.log({ name: e[0], count: e[1] });
    })
VM248:2 {name: 2, count: 3}
VM248:2 {name: undefined, count: undefined}
```

Reply:
@zhangloo333 Details should never be empty. That would mean something on the backend is broken or the tree parser is doing the wrong thing.

The `name` is allowed to be `undefined` if a nested entity is not given a name in YAML. The `count` will be still be present from the tree parser.